libs/search-widget/src/widgets/search-widget/SearchBar.svelte

```diff
@@ -47,6 +52,8 @@
   export let cssPath = '';
   export let prompt = '';
   export let no_tracking = false;
+  export let rag_strategies = '';
```

Comment:
should these options also be added to ChatWidget.svelte ?

Reply:
good question, I don't know but I'll ask @ebrehault